checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -27,3 +27,19 @@ class Config {
         return [Config]::new($use_rcc)
     }
 }
+function CreateCommand([Config]$config) {
+    if ($config.rcc) {
+        if (-Not (Test-Path -Path "./rcc.exe" -PathType Leaf)) {
+           $errorMessage = "Error: 'rcc.exe' binary not found in the current folder."
+            throw New-Object System.IO.FileNotFoundException -ArgumentList $errorMessage
+        }
+        # TODO: Add the arguments.
+        # What do I need?
+        # Simon is calculating the blueprint here
+        return ("./rcc.exe")
```

Comment:
Here, we can assume that we have a robot.yaml (let's say also in ../config) with a single task, so we want to execute something like
`rcc run --robot ../config`
This can be easily adapter later on.